server_admin/topics/authentication/webauthn.adoc

```diff
@@ -68,11 +68,10 @@ image:images/webauthn-browser-flow-conditional-with-OTP.png[]
 
 ==== Authenticate with WebAuthn Authenticator
 
-After registering their WebAuthn authenticator, the user carries out the following operations :
+After registering their WebAuthn authenticator, the user carries out the following operations
+assuming that authentication flow configuration above with the conditional subflow using `WebAuthn Authenticator` was used:
 
-- Open the login form.
-- Select the credential or credentials that {project_name} must accept for login. Then click `Save` and  click `Login`.
-- The list of registered Webauthn Authenticators' labels appears. Click `Authenticate`.
+- Open the login form. User must authenticate with username and password.
```

Comment:
Change "User must authenticate with"  to "The user must authenticate with a"

Reply:
Fixed